Framework/include/QualityControl/QualityObject.h

```diff
@@ -61,13 +63,13 @@ class QualityObject : public TObject
   /// @return the quality of the object
   ///
   void updateQuality(Quality quality);
-  Quality getQuality() const;
+  [[nodiscard]] Quality getQuality() const;
```

Comment:
My personal opinion is that these just pollute the code without any real benefits, if you think it is justified, I am fine with it.

Reply:
I think that I agree, I'll remove them.